pkg/virt-controller/watch/vm.go

```diff
@@ -2230,21 +2206,8 @@ func (c *VMController) enqueueVm(obj interface{}) {
 	c.Queue.Add(key)
 }
 
-func (c *VMController) getPatchFinalizerOps(oldFinalizers, newFinalizers []string) ([]string, error) {
-	joldFinalizers, err := json.Marshal(oldFinalizers)
-	if err != nil {
-		return nil, err
-	}
-
-	jnewFinalizers, err := json.Marshal(newFinalizers)
-	if err != nil {
-		return nil, err
-	}
-
-	return []string{
-		fmt.Sprintf(`{ "op": "test", "path": "/metadata/finalizers", "value": %s }`, joldFinalizers),
-		fmt.Sprintf(`{ "op": "replace", "path": "/metadata/finalizers", "value": %s }`, jnewFinalizers),
-	}, nil
+func (c *VMController) getPatchFinalizerOps(oldFinalizers, newFinalizers []string) ([]byte, error) {
```

Comment:
nit: does it make sense to keep this function? Maybe better to call `patch.PatchTestReplaceFinalizers` directly where needed?

Reply:
Done, thanks!